lambdas/delete-organisation/src/test.ts

```diff
@@ -0,0 +1,13 @@
+import {handler, LambdaInputEvent} from './index';
+
+(async function () {
+    const inputEvent: LambdaInputEvent = {
+        pathParameters: { 
+            organisationId: "db8c078f-62c4-4ff6-a689-b18de402b252"
+        }
+    };
+    const result = await handler(inputEvent);
```

Comment:
can we not do assertions in our lambda tests? i would expect something like `expect handler(inputEvent) to return status 200`

Reply:
It would also be good to create a dedicated organisation in the test run and not rely on an existing one